src/css/slider.css

```
@@ -0,0 +1,185 @@
+/* stylelint-disable no-descending-specificity */
+
+/* Variables - insert current slider value with JS in this variable to get a working color fill */
+
+:root {
+  --slider-percentage: 50%;
+}
+
+.a-slider {
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+/* Reset standard input styles */
+
+.a-slider > input[type='range'] {
+  appearance: none;
+  outline: none;
+  width: 100%;
+}
+
+/* Labels */
+
+.a-slider > label {
+  display: flex;
+  margin-bottom: 20px;
+  font: var(--font-secondary);
+  color: var(--font-color);
+}
+
+.a-slider--value {
+  margin-left: auto;
+  font-weight: 600;
+}
+
+.a-slider > input[type='range']:disabled ~ label {
+  color: var(--color-moon-200);
+}
+
+/* Chrome - always replicate styles for each browser, it's the only way :( */
+
+.a-slider > input[type='range']::-webkit-slider-runnable-track {
```

This is the first time I see an input range. 😬Sorry for the silly question:

Wouldn't it be easier to maintain if we group the browser-specific selectors by slider piece?

```suggestion
.a-slider > input[type='range']::-webkit-slider-runnable-track,
.a-slider > input[type='range']::-moz-range-track,
.a-slider > input[type='range']::-ms-track {
```

Even if the properties are not the same for all browsers, we could group the common ones and extend only the different props.

---

Unfortunately that type of selector does not accept the comma between them, like your suggestion :( that's why they're all repeated...

---

😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 

I had a little heart attack with this CSS limitation. 😝 

Thanks for the explanation. I didn't know that.